recipes/otsvm/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.1i" %}
+
+package:
+  name: otsvm
+  version: {{ version }}
+
+source:
+  fn: otsvm-{{ version }}.tar.gz
+  url: https://github.com/openturns/otsvm/archive/v{{ version }}.tar.gz
+  sha1: b03b35e0846b7e8b9d75d8fab96ba06baae3d8f1
+
+build:
+  number: 0
+  skip: true  # [py3k or win]
```

Comment:
It appears we do have openturns on [OSX](https://travis-ci.org/conda-forge/staged-recipes/jobs/135522322#L435).